kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, name, init_config, instances):
 
     def check(self, instance):
         """The main entrypoint of the check."""
+        self.log.debug("Found `zk_connect_str`. Running legacy Kafka Consumer check.")
```

Comment:
Not necessarily:
```
        if instance.get('zk_connect_str') is None:
            # bury the kafka version check under the zookeeper check because if zookeeper then we should immediately use
            # the legacy code path regardless of kafka version
            kafka_version = cls._determine_kafka_version(init_config, instance)
            if kafka_version >= (0, 10, 2):
                return super(KafkaCheck, cls).__new__(cls)
        return LegacyKafkaCheck_0_10_2(name, init_config, instances)
```
If you want that log line it shoud go in an else on the __new__ method of `KafkaCheck`

Reply:
I believe I tried that. Putting it under `__new__` of the new check logs it without the instance identifier. Assuming there are multiple instances (uncommon but could happen?) where `zk_connect_str` exists, it would be impossible to pinpoint which instance fell back to the legacy check.

Reply:
But then the log statement may also not be true as the code can fall back here if zk_connect_str is none but the detected version is < 0.12.2

Reply:
```suggestion
        self.log.debug("Running legacy Kafka Consumer check.")
```
I think we can leave just with "Running legacy Kafka Consumer check."